native_client/client.cc

```diff
@@ -233,6 +252,61 @@ ProcessFile(ModelState* context, const char* path, bool show_times)
   }
 }
 
+std::vector<meta_word>
+WordsFromMetadata(Metadata* metadata)
+{
+  std::vector<meta_word> word_list;
+
+  std::string word = "";
+  float word_start_time = 0;
+
+  // Loop through each character
+  for (int i=0; i<metadata->num_items; i++) {
+	MetadataItem item = metadata->items[i];
+	
+	// This is ugly but std::isspace is very particular about what characters it will accept.
+	// Not sure what the best solution is to support multiple languages. 
+	unsigned char character = static_cast<unsigned char>(*item.character); 
```

Comment:
I think this is fine for now, if more complicated cases show up they can be dealt with. I suspect any languages where this doesn't work would require changing all of this logic, not just the use of `std::isspace`.